### PR TITLE
[MBUILDCACHE-93] Command line configuration to skip saving in cache

### PR DESCRIPTION
**Pull request by kbuntrock**
Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [x] Make sure there is a [MBUILDCACHE JIRA issue](https://issues.apache.org/jira/browse/MBUILDCACHE) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[MBUILDCACHE-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MBUILDCACHE-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [x] You have run the [Core IT][core-its] successfully.

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

[core-its]: https://maven.apache.org/core-its/core-it-suite/

### CODE REVIEW

File: src/site/markdown/parameters.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ This document contains various configuration parameters supported by the cache e
 | `-Dmaven.build.cache.restoreGeneratedSources=(true/false)` | Do not restore generated sources and directly attached files                                                                             | Performance optimization                                                           |
 | `-Dmaven.build.cache.alwaysRunPlugins=<list of plugins>`   | Comma separated list of plugins to always run regardless of cache state. An example: `maven-deploy-plugin:*,maven-install-plugin:install` | Remote cache setup/tuning/troubleshooting                                          |
 | `-Dmaven.build.cache.skipCache=(true/false)`               | Skip looking up artifacts in caches. Does not affect writing artifacts to caches, disables only reading when set to `true`               | May be used to trigger a forced rebuild when matching artifacts do exist in caches |
+| `-Dmaven.build.cache.saveDisabled=(true/false)`            | Skip writing build result in caches. Does not affect reading from the cache.               | Configuring MR builds to benefits from the cache, but restricting writes to the `master` branch |
```
Comment (AlexanderAshitkin) [inline src/site/markdown/parameters.md:39]: I would go with `skipSave`, it seems to be a more conventional naming in Maven

**Review by AlexanderAshitkin — APPROVED** [review]

File: src/site/markdown/parameters.md (hunk 1 above)
Comment (kbuntrock) [reply src/site/markdown/parameters.md]: Works for me (it's even the title of my branch 😋).

I've updated my commit.

**Comment by kbuntrock** [comment]
@olamy: thank you :)

**Comment by jira-importer** [comment]
Resolve #333

**Comment by jira-importer** [comment]
Resolve #333

**Pull request merged**